Header header
string bit_rate
string link_quality
string signal
string noise